pyproject.toml

```diff
@@ -90,8 +98,13 @@ extras = test
 commands = python -m pytest {posargs}
 usedevelop = true
 
-[testenv:{py37-,py38-,py39-,py310-,py311-,py312-,}pyright]
-deps = pyright
-commands = pyright --verifytypes exceptiongroup
+[testenv:{py37-,py38-,py39-,py310-,py311-,py312-,}typing]
```

Comment:
You need to update the `labels` option above too if you rename `pyright` to `typing`, yes?

Reply:
Good catch, fixed.